examples/simulated/plot_riemannian_gaussian.py

```diff
@@ -34,15 +34,18 @@
 samples_1 = sample_gaussian_spd(n_matrices=n_matrices,
                                 mean=mean,
                                 sigma=sigma,
-                                random_state=random_state)
+                                random_state=random_state,
+                                sampling_method='rejection')
```

Comment:
I would introduce a value called "auto" for the sampling_method method that would default to "rejection" if dim == 2 and slice otherwise so you don't have to expose these options in the tutorial and it will just become faster for users without any code change